rust/rsc/src/rsc/add_job.rs

```diff
@@ -24,7 +24,7 @@ pub async fn add_job(
         created_at: NotSet,
         hash: Set(hash.clone().into()),
         cmd: Set(payload.cmd),
-        env: Set(payload.env.as_bytes().into()),
```

Comment:
This change is strange to me since we didn't change how `env` is represented but how `cmd` is represented but here we are changing `env`
Edit: I see that we did in fact change the payload type of `env` to match the entity type. So this makes sense.